hugegraph-dist/src/assembly/travis/build-report.sh

```diff
@@ -30,6 +30,11 @@ fi
 cd hugegraph-test
 mvn jacoco:dump@pull-test-data -Dapp.host=localhost -Dapp.port=$JACOCO_PORT -Dskip.dump=false
 cd ../
+
+if [[ ! -e "${TRAVIS_DIR}/jacococli.jar" ]]; then
+  wget -P "${TRAVIS_DIR}" https://gh.api.99988866.xyz/https://raw.githubusercontent.com/apache/hugegraph-doc/master/dist/server/jacococli.jar
```

Comment:
the download url is not stable？

Reply:
the download url should be stable, the url is download button trigger